Source/Fuse.Reactive.Expressions/UnaryOperator.uno

```diff
@@ -63,21 +73,32 @@ namespace Fuse.Reactive
 
 			protected override void OnNewData(IExpression source, object value)
 			{
-				OnNewOperand(value);
+				if (source == _uo.Operand) { _hasValue = true; _value = value; }
```

Comment:
please add some newlines here for readability...

Reply:
Yeah, that'd be my preference to, but this matches the preexisting form in Binary/Ternanry/QuaternarnryOperator that I didn't do.  I was trying to keep it looking exactly the same in case somebdoy ever realized how to merge all four of them into one class.